### ISSUE TITLE
Invalid catalog definition

### ISSUE THREAD

**Issue by michaelcarrano** [littlerobots/version-catalog-update-plugin#23]
Hi,

First off, thanks for making this plugin. Has made the migration to version catalog easier even if I have to make manual adjustments.

Just wanted to share some issues I have ran into. Depending on what dependencies you have, the name for it might not be valid, specifically, I ran into issues where dependencies ended in either `plugin` or `versions`.

```
org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, alias 'com-github-ben-manes-versions' is not a valid alias.

* Exception is:
java.lang.RuntimeException: org.gradle.api.InvalidUserDataException: Invalid catalog definition:
  - Problem: In version catalog libs, alias 'com-github-ben-manes-versions' is not a valid alias.
    
    Reason: It shouldn't end with 'versions'.
    
    Possible solution: Use a different alias which doesn't end with 'bundle', 'bundles', 'plugin', 'plugins', 'version' or 'versions'.
    
 ```
Looks like there are some reserved keywords that we should not be using, so wondering if there is a way to detect and provide an alternate naming? I'm not quite sure what the naming would look like.

**Comment by hvisser** [comment]
Yeah this is kind of tricky and these naming rules have been relaxt a bit for different Gradle versions, which version are you on? 

The naming strategy is pretty simple now and could probably be updated a bit to make sure none of the generated names end with these reserved words. The question is what to change in that case 🤔 

One solution could be just to drop the offending part, or if that is not possible (because it would clash or whatever) add `-generated` or something to it...wdyt?

**Comment by hvisser** [comment]
👍  Adding `-changeme` or `-generated` is probably the most pragmatic option for now, especially since this only happens when you use `--create` (or maybe also for new versions, but I'd have to check that)

**Comment by FyiurAmron** [comment]
I'd say that with Gradle 7.6 and 8.1 around and this problem being related to a release that's about a year old, it's more or less time to actually consider this a non-issue with the solution being simply "upgrade your Gradle or patch the file manually" :D

**Comment by hvisser** [comment]
Yep I agree!
